analyzers/src/SonarAnalyzer.Common/Helpers/MethodParameterLookupBase.cs

```diff
@@ -22,6 +22,7 @@ namespace SonarAnalyzer.Helpers
 {
     public interface IMethodParameterLookup
     {
+        bool TryGetSymbol(SyntaxNode argument, out IParameterSymbol parameter);
```

Comment:
Thanks for adding this. We need to touch this interface again but in another PR. I created https://github.com/SonarSource/sonar-dotnet/issues/6684 to track the work.

Reply:
Great!